src/redux/actions/follow.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import { FOLLOW, UNFOLLOW } from './actionTypes';
+
+const apiUrl = 'https://vidar-ah-backend-production.herokuapp.com/api/v1';
+
+export const followUser = userId => async (dispatch) => {
+  try {
+    const token = localStorage.getItem('token');
+
+    const { data } = await axios.post(`${apiUrl}/follow`, {
+      id: userId,
+    }, {
+      headers: {
+        'x-access-token': token,
+      },
+    });
+
+    dispatch({
+      type: FOLLOW
+
+    });
+
+    return data.message;
+  } catch (error) {
+    if (error.response && error.response.data.errors[0] === "You cannot follow a user twice.") {
```

Comment:
@olajide1234 why are we doing this? If we want a specific error message, we should just send it from the backend.

Reply:
Updated now.